resources/views/activity.blade.php

```diff
@@ -0,0 +1,90 @@
+@extends("layout.public")
+
+@section("content")
+    <div class="relative md:block -z-10">
+        <div class="absolute -left-[6%] md:left-[1%] top-28 md:top-40">
+            <x-bubble size="80"/>
+        </div>
+        <div class="absolute left-[8%] md:left-[4%] top-20 md:top-28">
+            <x-bubble size="30"/>
+        </div>
+        <div class="absolute right-[4%] top-36">
+            <x-bubble size="40"/>
+        </div>
+        <div class="relative top-24">
+            <div class="absolute right-[8%] md:right-[3%] top-72">
+                <x-bubble size="90"/>
+            </div>
+        </div>
+        <div class="relative top-24">
+            <div class="absolute hidden md:block md:right-[4%] top-96">
+                <x-bubble size="200"/>
+            </div>
+
+            <div class="absolute z-10 hidden lg:block -left-[10%] lg:-left-[3%] top-96">
+                <x-bubble size="250"/>
+            </div>
+        </div>
+    </div>
+    <section class="flex flex-col gap-16 xl:gap-32 lg:flex-row place-content-between relative mx-[10%] md:mx-[20%] lg:pt-32">
+        <div class="flex flex-col">
+            <div class="relative aspect-square max-w-xl overflow-hidden rounded-3xl order-2 lg:order-1 mt-8 md:mt-16 lg:mt-0">
+                <img src="{{  asset("storage/".$activity['photo']) }}"
+                     alt=""
+                     class="inset-0 -z-10 size-full object-cover">
+            </div>
+            <div class="mt-8 lg:mt-16 w-full max-w-2xl lg:max-w-none order-1 lg:order-2">
+                <div class="max-w-xl">
+                    <div class="text-2xl md:text-3xl lg:text-4xl">
+                        <h1 class="font-bold">{{ $activity['title']  }}</h1>
+                        <h1>{{ $activity['subtitle'] }}</h1>
+                    </div>
+
+                </div>
+            </div>
+            <div class="order-3">
+                <p class="mt-20 text-lg lg:text-xl font-light">{{ $activity['description'] }}</p>
+                @if(!empty( $activity['url'] ))
+                    <a class="block mt-10 text-lg lg:text-xl font-light"
+                       href="{{ $activity['url'] }}">▶ {{ $activity['url'] }}</a>
```

Comment:
```suggestion
                       href="{{ $activity['url'] }}">▶ {{ $activity['url'] }}</a>
```
why no `$activity->url`?